### PR TITLE
adding custom tomcat user for log rotate.

### PR DESCRIPTION
**Pull request by ackiejoe**

### CODE REVIEW

File: default.env.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+DATE=`date +%Y-%m-%d-%H-%M-%S`
+
+HOSTNAME=`hostname`
+USER=`whoami`
+
+yellow='\033[0;33m'
+green='\033[0;32m'
+red='\033[0;31m'
+NC='\033[0m'
```
Comment (adam-vessey) [inline default.env.sh:19]: Missing newline?

Comment (adam-vessey) [inline default.env.sh:19]: Strictly, almost thinking that this should be in some kind of separate library thing, if we don't expect it to change... could "source" it separately. Like, a `util.sh` or `lib.sh` or something, that could be sourced elsewhere in whatever file?

File: manual_install.sh [hunk 2]
```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
-DATE=`date +%Y-%m-%d-%H-%M-%S`
+if [ -f "env.sh" ]
+then
+  source env.sh
+else
+  echo -e "${red}Copy default.env.sh to env.sh and set accordingly${NC}"
```
Comment (adam-vessey) [inline manual_install.sh:7]: Can we expect `${red}` to be available, if the `env.sh` could not be loaded, given [it defines it](https://github.com/discoverygarden/islandora_log_config/pull/23/files#diff-bb1f8cfcb13f0582d148f53dc75e42fae5d8a4b006d00068c22ded49ac6d6c1cR18)?

File: default.env.sh (hunk 1 above)
Comment (ackiejoe) [reply default.env.sh]: fixed

Comment (ackiejoe) [reply default.env.sh]: yes all our scripts could benefit from a central lib and env file, out of scope for this.

File: manual_install.sh (hunk 2 above)
Comment (ackiejoe) [reply manual_install.sh]: fixed

**Pull request merged**